openedx/core/apidocs.py

```diff
@@ -10,7 +10,7 @@
     version="v1",
     description="APIs for access to Open edX information",
     #terms_of_service="https://www.google.com/policies/terms/",         # TODO: Do we have these?
-    email="oscm@edx.org",
+    email="api-support@edx.org",
```

Comment:
```suggestion
    email=settings.API_ACCESS_FROM_EMAIL,
```

I think this would be the appropriate Django setting?

Reply:
Good point, we can use the setting.  It's `api-requests@edx.org` for edX.